src/OrchardCore.Modules/OrchardCore.Spatial/Views/GeoPointField.Edit.cshtml

```diff
@@ -0,0 +1,25 @@
+@using OrchardCore.Spatial.Settings
+@using OrchardCore.Spatial.ViewModels
+@model EditGeoPointFieldViewModel
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<GeoPointFieldSettings>();
+}
+
+<div class="form-row">
+    <div class="form-group col-md-4">
+        <label asp-for="Latitude">Latitude</label>
+        <input asp-for="Latitude" class="form-control content-preview-text" required="@settings.Required" />
```

Comment:
It's a decimal right? so type="number" and need to apply the other fixes for localized decimal points. i.e. 56,0,0 vs 56.0.0, or just need to check how the script behaves in a different locale?

Reply:
Need to remember what we do for this. Probably need to look at the NumericField.

Reply:
The issue here is that it is a decimal without a length precision.

Reply:
Here is Standard representation of geographic point location by coordinates
https://www.iso.org/standard/39242.html

Reply:
I'm going to fix the values to 6 numbers after decimal point because over that it's a little too much.